ipalib/x509.py

```diff
@@ -488,6 +489,77 @@ def load_private_key_list(data, password=None):
     return priv_keys
 
 
+def load_certificates_from_file(filename):
```

Comment:
This is true. I considered abstracting this completely into two separate methods, one to retrieve all certs from a set of files and one to retrieve the set of keys and then let the independent functions iterate over them. This way, for example, load_certificates_from_files() could skip the key loading that it would never use.

I felt that in the end it didn't add a ton of value. If you disagree I can go ahead and do the abstraction.

Reply:
I fully second @frasertweedale on this one, this piece of codebase must not by be copy-pasted.

Reply:
I fully second @frasertweedale on this one, this piece of codebase must not by be copy-pasted.

Reply:
I fully second @frasertweedale on this one, this piece of codebase must not be copy-pasted.

Reply:
I fully second @frasertweedale on this one, this piece of codebase must not be copy-pasted.

Reply:
I fully second @frasertweedale on this one, this piece of codebase must not be copy-pasted.

Reply:
I fully second @frasertweedale on this one, this piece of codebase must not be copy-pasted.

Reply:
I fully second @frasertweedale on this one, this piece of codebase must not be copy-pasted.